web-service/helpers.py

```diff
@@ -189,12 +196,8 @@ def GetContentEncoding(content):
             encoding = value[0]
 
     if encoding is None:
-        try:
-            encoding = 'utf-8'
-            u_value = unicode(content, 'utf-8')
-        except UnicodeDecodeError:
-            encoding = 'iso-8859-1'
-            u_value = unicode(content, 'iso-8859-1')
+        encoding = 'iso-8859-1'
```

Comment:
Trying utf-8 encoding is done at the very beginning of `GetContentEncoding` function. That way, if a website is encoded using utf-8 but with a misconfigured charset, we can deliver right content.